cernopendata/modules/fixtures/data/docs/delphi-guide-analysis/delphi-guide-analysis.md

```diff
@@ -0,0 +1,271 @@
+# Running a DELPHI analysis job
```

Comment:
We don't need first-level heading, this will be inserted by the open data portal itself based on the docs name (that is configured in `delphi-guide-analysis.json`).